docs/cli/pyats_logs.rst

```diff
@@ -87,3 +87,43 @@ The following is an example of what the webpage looks like:
 
 Each tab shows the detailed results of each section of each testscript, as well
 as all commands executed and specific sections of logs.
+
+
+
+pyats logs upload
+---------------
+
+This subcommand will upload a zipped file to Taas Logviewer. It is an internal to 
+Cisco only feature.
+
+.. code-block:: text
+
+    Usage:
+      pyats logs upload [archive] [options]
+
+    Description:
+      Upload pyATS generated archive zip files to Taas Logviewer.
+
+      Examples:
+          # uploads an archive zip file to Taas Logviewer (default URL)
+          $ pyats logs upload file.zip
```

Comment:
$ pyats logs upload archive.zip 

Change that for all  instead of just file.zip

Reply:
fixed